packages/ui/src/composites/wui-card-select/index.ts

```diff
@@ -0,0 +1,43 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { globalStyles } from '../../utils/ThemeUtil'
+import '../../components/wui-text'
+import '../wui-wallet-image'
+import styles from './styles'
+
+@customElement('wui-card-select')
+export class WuiCardSelect extends LitElement {
+  public static styles = [globalStyles, styles]
+
+  // -- state & properties ------------------------------------------- //
+
+  @property() public imageSrc = ''
+
+  @property() public name = ''
+
+  @property({ type: Boolean }) public disabled = false
+
+  // -- render ------------------------------------------------------- //
+  public render() {
+    return html`
+      <button ?disabled=${this.disabled}>
+        <wui-wallet-image size="md" src=${this.imageSrc} alt=${this.name}></wui-wallet-image>
+        <wui-text variant="tiny-500" color="inherit">${this.getTruncatedName()}</wui-text>
+      </button>
+    `
+  }
+
+  private getTruncatedName() {
```

Comment:
This is a good candidate to make into utility inside ui package i.e. `HelperUtil` or `ToolsUtil`, you can reference how v2 does some of these files. Alternatively, can we acheive this with css? i.e https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

Reply:
Went for the `css` solution. Cleaner imo.